test/glossaries/glossaries.test.js

```diff
@@ -0,0 +1,204 @@
+/* eslint-disable no-unused-expressions */
+const mongoose = require("mongoose");
+const { describe, before, after, it } = require("mocha");
+
+const request = require("supertest");
+const { expect } = require("chai");
+const User = require("../../models/User");
+const Glossary = require("../../models/Glossary");
+const Keyword = require("../../models/Keyword");
+const app = require("../../app");
+
+describe("GET /glossaries?keyword=<WORD>&?page=<Number>?limit=<Number> test", function callback() {
```

Comment:
```suggestion
describe("GET /glossaries?keyword=<WORD>&page=<Number>&limit=<Number> test", function callback() {
```
이렇게 하는게 자연스러울 것 같습니다!

Reply:
네 반영하여 고쳤습니다!